### PR TITLE
Build updates for DSF Plugins

### PR DESCRIPTION
**Pull request by dbendele**
This PR adds automated build support on nijenkins for **UDP** and **GE Reflective Memory** plugins for the Data Sharing Framework custom device.

### CODE REVIEW

**Comment by dbendele** [comment]
\\nirvana\measurements\VeriStandAddons\data_sharing_framework_plugins\NI\export\PR-2\Build 7

File: Jenkinsfile [hunk 1]
```diff
@@ -2,7 +2,12 @@
 //Leave the above line alone.  It identifies this as a groovy script.
 @Library('vs-build-tools') _
 
-def lvVersions = ['2017', '2018', '2019']
+def lvVersions = ['2017', '2018', '2019', '2020']
 
-ni.vsbuild.PipelineExecutor.execute(this, 'lvmm', lvVersions)
+List<String> dependencies = ['niveristand-data-sharing-framework-custom-device']
+
+// Force niveristand-data-sharing-framework-custom-device to use an existing artifact, instead of being rebuilt.
```
Comment (dbendele) [inline Jenkinsfile:9]: REMOVE before merge

File: build.toml [hunk 2]
```diff
@@ -1,40 +1,80 @@
 [archive]
-build_output_dir = 'C:\Builds\Data Sharing Framework'
-archive_location = '\\nirvana\Measurements\VeriStandAddons\data_sharing_framework'
+build_output_dir = 'Components\Built'
+archive_location = '\\nirvana\Measurements\VeriStandAddons\data_sharing_framework_plugins'
 
-[projects.windows]
-path = 'Core\Data Sharing Framework Core Build - Windows.lvproj'
+[dependencies.niveristand-data-sharing-framework-custom-device]
+libraries = ['..\..\Custom Device\{target}\DSF Core.lvlibp']
```
Comment (dbendele) [inline build.toml:6]: REMOVE "..\..\" before merging this PR

File: control_dsf_plugins [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Package: ni-data-sharing-framework-plugins-support-veristand-{veristand_version}
```
Comment (buckd) [inline control_dsf_plugins:1]: The NIPM convention is to put "support" after the product.

`Package: ni-data-sharing-framework-plugins-veristand-{veristand_version}-support`

Comment (dbendele) [reply control_dsf_plugins]: aaa5091e45817ab0702f5b58822a53a43aef2c0c

File: build.toml (hunk 2 above)
Comment (dbendele) [reply build.toml]: aaa5091e45817ab0702f5b58822a53a43aef2c0c

File: Jenkinsfile (hunk 1 above)
Comment (dbendele) [reply Jenkinsfile]: 73968eeffd379bbbbc2a646ac767244e4efbb419

**Pull request merged**